client/settings/fraud-protection/advanced-settings/index.js

```diff
@@ -36,6 +36,38 @@ import './../style.scss';
 
 import { ProtectionLevel } from './constants';
 import { readRuleset, writeRuleset } from './utils';
+import wcpayTracks from 'tracks';
+
+const observerMapping = {
+	'avs-mismatch-card': {
```

Comment:
If all is going to have one single property, why don't we just rename it to `observerEventMapping` and get rid of the inner objects, just leave it with a key-value object?